File: API/Models/PermissaoPerfilListaModel.cs
namespace API_Visitatus.Models
{
    public class PermissaoPerfilListaModel
    {
        public int perCodi { get; set; }
        public string? perNome { get; set; }
        public bool perStat { get; set; }
        public int pemCodi { get; set; }
        public string? pemNome { get; set; }
        public bool pemStat { get; set; }
        public bool papAtvo { get; set; }
        public bool pepStat { get; set; }
    }
}

File: Sistema/src/app/models/PermissaoPerfilLista.Model .ts
export class PermissaoPerfilListaModel {
  perCodi: number;
  perNome: string;
  perStat: boolean;
  pemCodi: number;
  pemNome: string;
  pemStat: boolean;
  papAtvo: boolean;
  pepStat: boolean;

  constructor(
    perCodi: number,
    perNome: string,
    perStat: boolean,
    pemCodi: number,
    pemNome: string,
    pemStat: boolean,
    papAtvo: boolean,
    pepStat: boolean
  ) {
    this.perCodi = perCodi;
    this.perNome = perNome;
    this.perStat = perStat;
    this.pemCodi = pemCodi;
    this.pemNome = pemNome;
    this.pemStat = pemStat;
    this.papAtvo = papAtvo;
    this.pepStat = pepStat;
  }
}
